packages/visx-xychart/src/hooks/useDimensions.ts

```diff
@@ -7,29 +7,45 @@ const DEFAULT_DIMS = {
 };
 
 export type Dimensions = typeof DEFAULT_DIMS;
+type Margin = { top: number; right: number; bottom: number; left: number };
+export type PartialDimensions = { width?: number; height?: number; margin?: Partial<Margin> };
 
 /** A hook for accessing and setting memoized width, height, and margin chart dimensions. */
 export default function useDimensions(
-  initialDims?: Partial<Dimensions>,
-): [Dimensions, (dims: Dimensions) => void] {
+  initialDims: PartialDimensions = {},
+): [Dimensions, (dims: PartialDimensions) => void] {
   const [dimensions, privateSetDimensions] = useState<Dimensions>({
-    width: initialDims?.width == null ? DEFAULT_DIMS.width : initialDims.width,
-    height: initialDims?.height == null ? DEFAULT_DIMS.height : initialDims.height,
-    margin: initialDims?.margin == null ? DEFAULT_DIMS.margin : initialDims.margin,
+    // using || instead of ?? to include 0 as falsy
+    width: initialDims.width || DEFAULT_DIMS.width,
+    height: initialDims.height || DEFAULT_DIMS.height,
```

Comment:
With this change, I wonder if `DEFAULT_DIMS` should be updated to match the default in XYChart? If so, I think I can remove the `null` from the margin type and maintain current behavior.